packages/reframed/src/reframed.ts

```diff
@@ -77,10 +87,32 @@ export function reframed(
 	//       has added the various load event listeners
 	document.body.insertAdjacentElement("beforeend", iframe);
 
+	const ready = Promise.all([monkeyPatchReady, reframeReady]).then(
+		([cleanup]) => {
+			reframedContainer.shadowRoot.reframeMetadata.iframeDocumentReadyState =
+				"interactive";
+			reframedContainer.shadowRoot!.dispatchEvent(
+				new Event("readystatechange")
+			);
+			// TODO: this should be called when reframed async loading activities finish
+			//        (note: the 2s delay is completely arbitrary, this is a very temporary solution anyways)
+			//       (see: https://github.com/web-fragments/web-fragments/issues/36)
+			setTimeout(() => {
+				reframedContainer.shadowRoot.reframeMetadata.iframeDocumentReadyState =
+					"complete";
+				reframedContainer.shadowRoot.dispatchEvent(
+					new Event("readystatechange")
+				);
+			}, 2_000);
```

Comment:
Do we have to set an actual timeout. Can we set the timeout to 0?

Reply:
I worry that we'll hit temporal issues quite easily if timeout is 0. 2s is not ideal, but will delay urgency of fixing #36, which I think is preferable right now.